packages/css-jss/src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+export type Style = {} | null | void | string
```

Comment:
you should be able to use https://github.com/cssinjs/jss/blob/master/packages/jss/src/index.d.ts#L4

Reply:
Thank you, updated code